var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -0,0 +1,80 @@
+##############################################################################
+# Copyright (c) 2017, Innovative Computing Laboratory
+# Produced at the Innovative Computing Laboratory.
+#
+# Created by Piotr Luszczek, luszczek@icl.utk.edu, All rights reserved.
+#
+# For details, see https://github.com/llnl/spack
+#
+##############################################################################
+#
+from spack import *
+
+
+class Plasma(MakefilePackage):
+    """Parallel Linear Algebra Software for Multicore Architectures, PLASMA is
+    a software package for solving problems in dense linear algebra using
+    multicore processors and Xeon Phi coprocessors. PLASMA provides
+    implementations of state-of-the-art algorithms using cutting-edge task
+    scheduling techniques. PLASMA currently offers a collection of routines for
+    solving linear systems of equations, least squares problems, eigenvalue
+    problems, and singular value problems."""
+
+    homepage = "https://bitbucket.org/icl/plasma/"
+    url      = "https://bitbucket.org/icl/plasma/downloads/plasma-17.1.tar.gz"
+
+    version("17.1", "64b410b76023a41b3f07a5f0dca554e1")
+
+    version("develop", hg="https://luszczek@bitbucket.org/icl/plasma")
+
+    depends_on("blas")
+    depends_on("lapack")
+
+    # Intel's MKL installer accesses /opt regardless of PREFIX
```

Comment:
out of curiosity: can you elaborate when/why intel-mkl access /opt?

@lee218llnl have you heard anything like that about Intel packages?

Reply:
Yeah, I am confused by the comment, too!

Reply:
I'm not sure if this is the issue, but if you have previously installed Intel components in /opt, then the Intel Installer will force subsequent installations in there too. I generally work around this by removing the $HOME/intel/intel_sdp_products.db file that the installer cues off of.

Reply:
I was hunting for the log file. But after "spack uninstall intel-mkl" it disappeared. I remember Spack complained that the "install" command returned non-zero value and the Intel Installer log mentioned "no access to /opt". But this is as much as my memory is worth.

I've just tried to install MKL 2018 and it worked. I have "remotemon*.db" files in ~/intel/ism/rm but no errors.

I'm trying a PLASMA build with MKL right now. It gets tricky to switch between OpenBLAS and MKL inside a single package but I'll give it a try and push again.